### PR TITLE
waveshare2in13v2: Add constants, add missing init to example

### PR DESCRIPTION
**Pull request by hansmi**
Translate magic numbers to named constants based on datasheet and add missing calls to `dev.Init` in example code.

### CODE REVIEW

File: waveshare2in13v2/example_test.go [hunk 1]
```diff
@@ -73,7 +77,11 @@ func Example_other() {
 
 	dev, err := waveshare2in13v2.NewHat(b, &waveshare2in13v2.EPD2in13v2) // Display config and size
 	if err != nil {
-		log.Fatalf("failed to initialize epd: %v", err)
+		log.Fatalf("Failed to initialize driver: %v", err)
+	}
+
+	if err := dev.Init(false); err != nil {
```
Comment (maruel) [inline waveshare2in13v2/example_test.go:83]: Please unshadow.

File: waveshare2in13v2/waveshare213v2.go [hunk 2]
```diff
@@ -21,6 +20,32 @@ import (
...
+	displayUpdateControl1          byte = 0x21
+	displayUpdateControl2          byte = 0x22
+	writeRAMBW                     byte = 0x24
+	writeRAMRed                    byte = 0x26
+	writeVcomRegister              byte = 0x2C
+	writeLutRegister               byte = 0x32
+	setDummyLinePeriod             byte = 0x3A
+	setGateTime                    byte = 0x3B
+	borderWaveformControl          byte = 0x3C
+	unknownCommand37               byte = 0x37
```
Comment (maruel) [inline waveshare2in13v2/waveshare213v2.go:40]: Was it listed in the datasheet or not?

File: waveshare2in13v2/example_test.go (hunk 1 above)
Comment (hansmi) [reply waveshare2in13v2/example_test.go]: Done.

File: waveshare2in13v2/waveshare213v2.go (hunk 2 above)
Comment (hansmi) [reply waveshare2in13v2/waveshare213v2.go]: Ah, this was a leftover from an earlier revision. No, this particular comment isn't documented anywhree. I removed the constant again.

**Comment by maruel** [comment]
Thanks!

**Pull request merged**